### PR TITLE
Junk/typing test

### PR DESCRIPTION
**Pull request by jibrang**
#### What does this PR do?
- Update the components for correctly display a typing gif where it is requested

#### Where should the reviewer start?
src/History.tsx
src/ActivityView.tsx

### CODE REVIEW

File: botchat-fullwindow.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@media (max-width: 450px) {
+  .wc-card {
+    border: 1px solid #d2dde5;
+    padding: 8px;
+    width: 198px; } }
```
Comment (isa-bel) [inline botchat-fullwindow.css]: Can you put the `}` in their own line?

File: src/ActivityView.tsx [hunk 2]
```diff
@@ -50,27 +53,36 @@ export class ActivityView extends React.Component<ActivityViewProps, {}> {
         return this.props.activity !== nextProps.activity || this.props.format !== nextProps.format;
     }
 
+    componentWillMount() {
+        const { activity, switchTyping } = this.props;
+
+        switchTyping(!!activity.custom_typing);
```
Comment (isa-bel) [inline src/ActivityView.tsx]: is better to use `Boolean(activity.custom_typing)`

File: src/Chat.tsx [hunk 3]
```diff
@@ -45,7 +46,7 @@ export class Chat extends React.Component<ChatProps, {}> {
 
     constructor(props: ChatProps) {
         super(props);
-
+      window['display_typing'] = false;
```
Comment (isa-bel) [inline src/Chat.tsx]: Fix alignment please.

File: src/History.tsx [hunk 4]
```diff
@@ -20,13 +20,28 @@ interface Props {
...
 }
 
-class HistoryContainer extends React.Component<Props, {}> {
+export interface State {
+    isActiveTyping:boolean
+}
+
+class TypingDots extends React.Component<{}, {}> {
+    render() {
+      return (<div className="wave">
```
Comment (isa-bel) [inline src/History.tsx]: Could we get better classnames for this and the next? Maybe `typing-dots__wave`

File: src/History.tsx [hunk 5]
```diff
@@ -113,12 +129,14 @@ class HistoryContainer extends React.Component<Props, {}> {
                             onImageLoad={ () => this.autoscroll() }
                         />
                     ) }
+                    { this.state.isActiveTyping ? <TypingDots /> : null }
```
Comment (isa-bel) [inline src/History.tsx]: You can use `{ this.state.isActiveTyping && <TypingDots /> }` instead

File: static/botchat.css [hunk 6]
```diff
@@ -51,7 +51,7 @@ body .wc-app,
   font-size: 14px;
   vertical-align: middle;
   background-color: #ff633f;
-	border: solid 1px #e63109 !important;
+  border: solid 1px #e63109 !important;
```
Comment (isa-bel) [inline static/botchat.css]: Do we need this `!important`?

File: static/botchat.css [hunk 7]
```diff
@@ -212,201 +212,208 @@ body .wc-app,
...
+  height: 20px;
+  width: 64px;
+  }
+
+  @keyframes animationFrames {
   0% {
-    /*max-height: 0;*/
-    opacity: 0;
+  /*max-height: 0;*/
+  opacity: 0;
```
Comment (isa-bel) [inline static/botchat.css]: Why the change of indentation?

File: static/botchat.css [hunk 8]
```diff
@@ -212,201 +212,208 @@ body .wc-app,
...
   }
   100% {
-    /*max-height: 2000px;*/
+  /*max-height: 2000px;*/
+  }
   }
-}
 
-.wc-message {
+  .wc-message {
```
Comment (isa-bel) [inline static/botchat.css]: Why the change of indentation?

File: botchat-fullwindow.css (hunk 1 above)
Comment (jibrang) [inline botchat-fullwindow.css]: Yes, I will fix it

File: src/ActivityView.tsx (hunk 2 above)
Comment (jibrang) [inline src/ActivityView.tsx]: Ok, I  will change it

File: src/Chat.tsx (hunk 3 above)
Comment (jibrang) [inline src/Chat.tsx]: I will remove this line, because I don't need it

File: src/History.tsx (hunk 4 above)
Comment (jibrang) [inline src/History.tsx]: Yes, I will change it

File: src/History.tsx (hunk 5 above)
Comment (jibrang) [inline src/History.tsx]: Ok, I will change it

File: static/botchat.css (hunk 6 above)
Comment (jibrang) [inline static/botchat.css]: I am not sure, I will check it and if not, I will remove it

File: static/botchat.css (hunk 7 above)
Comment (jibrang) [inline static/botchat.css]: I used a command to indent all the file

File: static/botchat.css (hunk 8 above)
Comment (jibrang) [inline static/botchat.css]: I used a command to indent all the file

Comment (isa-bel) [inline static/botchat.css]: I guess there is some CSS that the command couldn't handle, because the resulting CSS is not correctly formatted:
```
  .wc-message {
  max-width: 80%;
  position: relative;
  }
```
See here https://github.com/wizeline/wizeline-bot-webchat/blob/5937fe85dec282df820881bd446fe1fda2be4ca1/static/botchat.css#L220

File: src/ActivityView.tsx (hunk 2 above)
Comment (jibrang) [inline src/ActivityView.tsx]: BTW, I read in AirB&B style guide that this is a good practice, but what do you think?
https://github.com/airbnb/javascript#coercion--booleans

File: static/botchat.css (hunk 6 above)
Comment (MoisesFlores22) [inline static/botchat.css]: yes it is important :trollface:

File: static/botchat.css (hunk 8 above)
Comment (jibrang) [inline static/botchat.css]: Yes, you are right, I found the issue, I will fix it

File: static/botchat.css (hunk 6 above)
Comment (isa-bel) [inline static/botchat.css]: :woman_facepalming:

Comment (isa-bel) [inline static/botchat.css]: 😢

File: src/ActivityView.tsx (hunk 2 above)
Comment (isa-bel) [inline src/ActivityView.tsx]: It's more explicit to use `Boolean()`, but if this project is using strict Airbnb, then use that.
